cddl/concise-tl-tag.cddl

```diff
@@ -0,0 +1,6 @@
+concise-tl-tag = {
```

Comment:
Warning: bikeshedding comment.

I'd prefer `concise-tag-list` because I'm not sure that CoTL is a "tag" in the sense that other Co{MID,SWID,TS} are.

Reply:
This would then beak the convention (we have as of today) that CoRIM contains a list of `tags`!

Reply:
I'm also OK with adding a new code point for `concise-tag-list` to `corim-map` as I agree with Thomas that I don't think these are the same type thing as comid, coswid, etc.
```
corim-map = {
...
? &(tag-list: 6) => [ + concise-tl-tag ]
...
}
```

Reply:
Lets separate two discussions.
1. Rename CoBOM to CoTL --> This is addressed here
2. Discuss whether CoTIL is a tag in itself or another data structure ? Discussed via Issue #372